pages/1.11/release-notes/1.11.3/index.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.11.3
+title: Release Notes for 1.11.3
+menuWeight: 0
+excerpt: Release notes for DC/OS 1.11.3
+---
+
+DC/OS 1.11.3 is released on June 26, 2018.
```

Comment:
*was* released.

Reply:
@seth I'll change the "tense (is >> was) after the release is done.